browser/brave_content_browser_client.cc

```diff
@@ -254,29 +254,21 @@ BraveContentBrowserClient::GetNavigationUIData(
   return std::move(navigation_ui_data);
 }
 
-std::unique_ptr<base::Value>
-BraveContentBrowserClient::GetServiceManifestOverlay(base::StringPiece name) {
-  auto chrome_overlay =
-    ChromeContentBrowserClient::GetServiceManifestOverlay(name);
-
-  ui::ResourceBundle& rb = ui::ResourceBundle::GetSharedInstance();
-  int id = -1;
-  if (name == content::mojom::kBrowserServiceName)
-    id = IDR_BRAVE_CONTENT_BROWSER_MANIFEST_OVERLAY;
-  else if (name == content::mojom::kPackagedServicesServiceName)
-    id = IDR_BRAVE_CONTENT_PACKAGED_SERVICES_MANIFEST_OVERLAY;
-  else
-    return chrome_overlay;
 
-  base::StringPiece manifest_contents =
-      rb.GetRawDataResourceForScale(id, ui::ScaleFactor::SCALE_FACTOR_NONE);
+#include "brave/components/services/brave_content_browser_manifest_overlay.h"
```

Comment:
Any reason these can't go up top?

Reply:
nope, good catch, I was just testing it originally then forgot to move it back.